### DISCUSSION TITLE
have a difficulty in modeling

### DISCUSSION THREAD

**Discussion by neperchh** [neperfepx/neper#744] [Q&A] [answered]
Dear teacher, I would like to ask about modeling. First of all, can neper construct a model in a specified range, such as x=0-2, y=1-2, if possible, can you give me a simple demonstration code. Secondly, why are the input seed points and the output seed points not completely consistent? How to make them exactly the same. thanks.
![image](https://github.com/neperfepx/neper/assets/127730078/ed713565-43aa-4b65-b957-dcb50d40ce2f)
![image](https://github.com/neperfepx/neper/assets/127730078/3c2651de-fb36-4a1a-a700-2c0b59272106)

**Comment by rquey** [comment, marked as answer]
Next time, please copy your files as text.

The index on the first column is the problem.  Remove this column and it'll work.
